### DISCUSSION TITLE
Qualquer componente Alcinoe que coloco no Form quando vou compilar aparece esse erro abaixo

### DISCUSSION THREAD

**Discussion by edugdr** [MagicFoundation/Alcinoe#370] [General]
![erro](https://github.com/user-attachments/assets/fd40b94b-51bf-4243-b119-6d3494b2f62e)
Estou usando o Delphi 12.2
Desde já agradeço

**Comment by Zeus64** [comment]
Você deve primeiro executar <Alcinoe>\CompileAll.bat para recuperar e corrigir localmente o código-fonte original do Delphi. Em seguida, você deve incluir no caminho de pesquisa os arquivos localizados em Alcinoe\Embarcadero\Athens\.

**Comment by edugdr** [comment]
Obrigado por responder, fiz o que escreveu acima mas da alguns erros quando executo o CompileAll.bat e já adicionei o path
![erro2](https://github.com/user-attachments/assets/173a47de-6156-4019-a1fa-15287aabd096)
![erro3](https://github.com/user-attachments/assets/4f386056-8dee-4ac5-afe8-d0cbdecfdac6)

**Comment by Zeus64** [comment]
Você pode acessar ou criar este diretório:
D:\Componentes\Delphi\Alcinoe-master\Libraries\Ios\Firebase\FirebaseAnalyticsSwift.FirebaseAnalyticsSwift.xcframework\ios-arm64_x86_64-maccatalyst\FirebaseAnalyticsSwift.framework\Modules\FirebaseAnalyticsSwift.swiftmodule\Project\arm64-apple-ios-macabi.swiftsourceinfo

Parece que seu sistema não consegue lidar com caminhos com mais de 255 caracteres. :(

**Comment by edugdr** [comment]
Estranho formatei o pc hoje, instalei o Delphi 12.2 mas apresenta esses erros

**Comment by Zeus64** [comment]
Não faço ideia, mas parece que seu PC não consegue lidar com caminhos com mais de 255 caracteres.

**Comment by edugdr** [comment]
Apaguei a pasta Alcinoe agora passou aquele erro mas apresentou esse
![erro4](https://github.com/user-attachments/assets/3b6fbe6d-f5ba-48a8-bec4-0eec5b8cea7e)

**Comment by Zeus64** [comment]
Mesmo problema com o limite de caminho

**Reply by edugdr** [reply]
Pode ser pq é o Delphi 12.2 professional e não o Rad ? Estranho que instala todos os componentes mas quando vai compilar da erro,  queria tanto testar esse componentes no Android.

**Reply by Zeus64** [reply]
Não, acho que é na configuração do seu Windows.

**Reply by edugdr** [reply]
Opa resolvi o problema dos 255 caracteres alterando nas variáveis de ambiente o path do delphi, agora passou o MSBuild porém está dando erro no run tests
![ERRO5](https://github.com/user-attachments/assets/5ce0e802-36e7-4d8c-8c94-e1b0802676b1)

**Comment by edugdr** [comment]
Boa Tarde Zeos64  Instalei em outro pc e apresenta o mesmo problema error E2003: Undeclared indentifier : Material.
No delphi todos os componentes Alcionoe estão instalados, porém se colocar qualquer um no form apresenta o mesmo erro acima na compilação.
Desculpa de incomodar mas já tentei várias vezes e nada.
Desde já agradeço.

**Comment by Zeus64** [comment]
Qual erro você tem ao executar o compileAll.bat?

**Comment by edugdr** [comment]
Boa Noite segue abaixo a imagem rodando o compileAll.bat
![ERRO5](https://github.com/user-attachments/assets/8266a6d9-a7f1-4ca4-8ffb-116a87e2005a)

**Comment by Zeus64** [comment]
Você pode tentar não executar o teste?

**Comment by edugdr** [comment]
Fiz aqui sem rodar o teste passou, quando chegou no build demos coloquei que sim ai apresenta os mesmos erro na imagem acima.
Rodei de novo sem fazer o build demos e foi até o fim, porém quando vai compilar no Delphi apresenta os erros acima.
![ERRO6](https://github.com/user-attachments/assets/792629e0-310b-4580-8425-931f383a11e3)

**Comment by Zeus64** [comment]
O git.exe deve estar no caminho do Windows e, ao copiar o código-fonte do Delphi localmente com o compileALL.bat, o git precisa estar presente para aplicá-lo.

**Comment by edugdr** [comment]
Bom dia opa instalei o git agora foi, muito obrigado mesmo
